### PR TITLE
extracted cfda load

### PR DESCRIPTION
**Pull request by sharb**
This is the new PR to extract cfda load.

### CODE REVIEW

File: dataactvalidator/scripts/load_cfda_data.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+             "length_and_time_phasing_of_assistance_(102)": "length_and_time_phasing_of_assistance",
+             "reports_(111)": "reports",
+             "audits_(112)": "audits",
+             "records_(113)": "records",
+             "account_identification_(121)": "account_identification",
+             "obligations_(122)": "obligations",
+             "range_and_average_of_financial_assistance_(123)": "range_and_average_of_financial_assistance",
+             "program_accomplishments_(130)": "program_accomplishments",
+             "regulations__guidelines__and_literature_(140)": "regulations_guidelines_and_literature",
+             "regional_or_local_office_(151)": "regional_or_local_office",
```
Comment (dpb-bah) [inline dataactvalidator/scripts/load_cfda_data.py]: Some bugs were addressed/fixed in a recently merged PR:
https://github.com/fedspendingtransparency/data-act-broker-backend/pull/1240

File: dataactvalidator/scripts/loaderUtils.py [hunk 2]
```diff
@@ -91,6 +91,6 @@ def clean_data(data, model, field_map, field_options):
 def format_date(value):
     """ Format date from 'MMM dd, yyyy' to 'yyyymmdd' """
 
-    formatted_value = pd.to_datetime(value, format="%b, %d %Y")
+    formatted_value = pd.to_datetime(value, format="%b %d,%Y")
```
Comment (kwhickey) [inline dataactvalidator/scripts/loaderUtils.py]: Is this date format something you changed? Something from a merge? Why did it need to change?

File: dataactvalidator/scripts/loadFile.py [hunk 3]
```diff
@@ -141,7 +141,7 @@ def load_cfda_program(filename):
              "range_and_average_of_financial_assistance_(123)": "range_and_average_of_financial_assistance",
              "program_accomplishments_(130)": "program_accomplishments",
              "regulations__guidelines__and_literature_(140)": "regulations_guidelines_and_literature",
-             "regional_or_local_office_(151)": "regional_or_local_office",
+             "regional_or__local_office_(151)": "regional_or_local_office",
```
Comment (kwhickey) [inline dataactvalidator/scripts/loadFile.py]: This looks suspicious. When you merge down from dev, make sure to check that this is the properly named thing. Or check without whoever changed it on another branch to ensure it was supposed to be changed.

File: dataactvalidator/scripts/load_cfda_data.py (hunk 1 above)
Comment (kwhickey) [reply dataactvalidator/scripts/load_cfda_data.py]: Yes. You'll need to be sure to merge that branch down (from wherever those fixes were made -- make sure they have been merged up to the target branch you're merging into). Then resolve any merge conflicts, from having removed the code that was edited/fixed, then re-copy the fixed code into your new file. (or do the same in a shorter way -- e.g. just copy the code from that branch and paste it into your file).

File: dataactvalidator/scripts/loaderUtils.py (hunk 2 above)
Comment (dpb-bah) [reply dataactvalidator/scripts/loaderUtils.py]: Correct. This was necessary for the CFDA loader.

File: dataactvalidator/scripts/loadFile.py (hunk 3 above)
Comment (dpb-bah) [reply dataactvalidator/scripts/loadFile.py]: Correct. This was necessary for the CFDA loader.

**Pull request merged**